backend/app/api/v1/health.py

```
@@ -0,0 +1,85 @@
+            "status": bot_status
+        }
+    except Exception as e:
+        logger.error(f"Discord bot health check failed: {e}")
+        raise HTTPException(
+            status_code=503,
+            detail={
+                "service": "discord_bot",
+                "status": "unhealthy",
+                "error": str(e)
+            }
+        )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)